packages/flame/test/geometry/polygon_component_test.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flame/components.dart';
+import 'package:flame/geometry.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('PolygonComponent', () {
```

Comment:
Could you move this to shape_component_test where the other tests for this are?
There are some inconsistencies of which directories the tests live currently so it can be quite hard to find the correct place. :sweat_smile: